pkgs/http/lib/src/utils.dart

```diff
@@ -74,3 +74,8 @@ Stream<T> onDone<T>(Stream<T> stream, void Function() onDone) =>
       sink.close();
       onDone();
     }));
+
+/// On upload progress callback.
```

Comment:
(Would drop the "On" here, just "Upload progress callback", it's cleaner. The upload progress callback is called on upload progress.)

Reply:
typedef removed